tools/sdk-generation-pipeline/Dockerfile

```diff
@@ -31,7 +31,7 @@ RUN apt install python3-pip -y && apt install python3-venv -y && pip3 install --
 RUN add-apt-repository ppa:deadsnakes/ppa -y && apt-get install python3.10 -y && apt-get install python3.10-venv -y
 
 # install powershell
-RUN wget -q https://packages.microsoft.com/config/ubuntu/20.04/packages-microsoft-prod.deb
+RUN wget -q https://packages.microsoft.com/config/ubuntu/22.04/packages-microsoft-prod.deb
```

Comment:
There is additional context for this change here:
- https://github.com/Azure/azure-sdk-tools/pull/4996#discussion_r1052977794

Reply:
Please revert this change because it's in our docker image, which is based on ubuntu 20.04.

Reply:
@dw511214992 is there a way for us to update it to ubuntu 22.04? What steps it would be necessary for that?

Thanks!

Reply:
@dw511214992 we are trying to move most of our agents to run on 22.04 so we should please take a tracking issue to update these to 22.04.

Reply:
@dw511214992 kindly asking for input on this :)

Reply:
I created a follow-up issue. Please see this comment:
- https://github.com/Azure/azure-sdk-tools/pull/5004#issuecomment-1398536891